pkg/pool-weighted/contracts/CustomFeeAuthorizer.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+
+contract CustomFeeAuthorizer {
+    event feeSetterAdded(address indexed feeSetter);
+    event feeSetterRemoved(address indexed feeSetter);
+
+    mapping(address => bool) private isCustomFeeSetter;
+    bool public isCustomFeeEnabled = false ;
```

Comment:
Safe to say solvers will always want this to be true. If we could we make this immutable, does the code become more efficient?